### DISCUSSION TITLE
[Support]: Frigate 0.14 compatibility with HomeAssistant and HACS

### DISCUSSION THREAD

**Discussion by rickgitdone** [blakeblackshear/frigate#13443] [General]
Before I update my standalone docker version to 0.14 I wanted to see if there are any compatibility issues with the current HACS and HA versions.  I have HA ver 2024.8.3 and HACS is on v 1.34.0 

0.13.2-6476f8a  Frigate ver

Firefox ver 129.0.2

Home Assistant OS ver 13.1 (OVA)
Home Assistant OS Open Virtual Appliance

**Comment by NickM-27** [comment]
no, there are no issues
